osu.Game/Online/Chat/ChatBase.cs

```diff
@@ -1,44 +1,25 @@
-// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Newtonsoft.Json;
 using osu.Framework.Configuration;
 using osu.Framework.Lists;
 
 namespace osu.Game.Online.Chat
 {
-    public class Channel
+    public abstract class ChatBase

```

Comment:
this naming is wrong

Reply:
Just naming it chat would force me to use chat.chat everywhere

Reply:
It's not `Chat`, it's a `Channel`